src/webview/helm-chart/app/cardItem.tsx

```diff
@@ -229,100 +294,47 @@ export class CardItem extends React.Component<DevFileProps, {
                 id={`modal-${selectedVersion.name}`}>
                 <div className={this.props.cardItemStyle.helmCardHeader}>
                     <div className={this.props.cardItemStyle.devPageCardHeader}>
-                        <div className={this.props.cardItemStyle.devPageTitle}>
-                            <img
-                                data-testid='icon'
-                                src={this.state.selectedVersion.icon ? this.state.selectedVersion.icon : require('../../../../images/helm/helm.svg').default}
-                                alt={this.state.selectedVersion.icon + ' logo'}
-                                className={this.props.cardItemStyle.cardImage}
-                                style={{ margin: '0rem' }} />
-                            <div style={{ padding: '1rem', margin: '0rem' }}>
-                                <Typography variant='subtitle1'>
-                                    {
-                                        capitalizeFirstLetter(this.props.helmEntry.displayName)
-                                    }
+                        <div style={{ display: 'flex', flexDirection: 'row', width: '100%' }}>
+                            <div className={this.props.cardItemStyle.devPageTitle} style={{ width: '50%', gap: '2rem' }}>
+                                <img
+                                    data-testid='icon'
+                                    src={this.state.selectedVersion.icon ? this.state.selectedVersion.icon : require('../../../../images/helm/helm.svg').default}
+                                    alt={this.state.selectedVersion.icon + ' logo'}
+                                    className={this.props.cardItemStyle.cardImage}
+                                    style={{ margin: '0rem' }} />
+                                <div style={{ margin: '0rem' }}>
+                                    <Typography variant='subtitle1'>
+                                        {
+                                            capitalizeFirstLetter(this.props.helmEntry.displayName)
+                                        }
+                                    </Typography>
+                                </div>
+                            </div>
+                            <div style={{ display: 'flex', flexDirection: 'row', gap: '15rem' }}>
+                                <Typography variant='body1'>
+                                    Provider: {selectedVersion.annotations['charts.openshift.io/providerType']}
```

Comment:
@msivasubramaniaan Let's remove this. And directly add `Provided by` company name. Same as we have in card information.

Reply:
@msivasubramaniaan Let's remove this. And directly add `Provided by` company name. Same as we have in card information.